src/fu-util-common.c

```diff
@@ -1610,9 +1616,26 @@ fu_util_security_attrs_to_string (GPtrArray *attrs)
 				FwupdSecurityAttr *attr = g_ptr_array_index (attrs, i);
 				if (!fwupd_security_attr_has_flag (attr, hpi_suffixes[j]))
 					continue;
+				if (fwupd_security_attr_has_flag (attr, FWUPD_SECURITY_ATTR_FLAG_RUNTIME_ISSUE) &&
+				    !fwupd_security_attr_has_flag (attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS))
+					runtime_help = TRUE;
 				fu_security_attr_append_str (attr, str);
 			}
 		}
 	}
+
+	if (low_help) {
+		/* TRANSLATORS:  this is instructions on how to improve the HSI security level */
+		g_string_append_printf (str, "\n%s %s",
+					_("Your system has a low HSI security level. Visit this URL for more information: "),
+					"https://github.com/fwupd/fwupd/wiki/Low-host-security-level");
+	}
+	if (runtime_help) {
+		/* TRANSLATORS:  this is instructions on how to improve the HSI suffix */
```

Comment:
frustratingly, the TRANSLATORS line has to be the line below, it only gets found if it's one logical line above the _()

Reply:
wow, that's annoying.  OK fixed.